File: ProjectArena.Infrastructure/Models/Battle/Synchronization/SpecEffectDto.cs
namespace ProjectArena.Infrastructure.Models.Battle.Synchronization
{
    public class SpecEffectDto
    {
        public int Id { get; set; }

        public string OwnerId { get; set; }

        public bool IsAlive { get; set; }

        public int X { get; set; }

        public int Y { get; set; }

        public float Z { get; set; }

        public float? Duration { get; set; }

        public string NativeId { get; set; }

        public string Visualization { get; set; }
    }
}

File: ProjectArena.Bot/Models/Dtos/SpecEffectDto.fs
namespace ProjectArena.Bot.Models.Dtos

type SpecEffectDto = {
    Id: int
    OwnerId: string option
    IsAlive: bool
    X: int
    Y: int
    Z: float32
    Duration: float32 option
    NativeId: string
    Visualization: string
}